provider/README.md

```diff
@@ -95,6 +95,7 @@ Data: `{ "trips": [] }`, an array of objects with the following structure
 | `trip_duration` | Integer | Required | Time, in Seconds |
 | `trip_distance` | Integer | Required | Trip Distance, in Meters |
 | `route` | Route | Required | See detail below |
+| `access_method` | Enum | Required | How the user gained access to the vehicle |
```

Comment:
Can you link to `#access-methods` here like we do for the other enum fields/tables? 

Otherwise this looks great, and agreed it is a useful datapoint to have.